DESCRIPTION

```diff
@@ -4,81 +4,65 @@ Title: Identify oncogenes and tumor suppressor genes from omics data
 Version: 2.0.0
 Date:
 Authors@R:
-  c(person("Antonio", "Colaprico",
-           role="aut"),
-    person("Catharina", "Olsen",
-           role="aut"),
-    person("Matthew H.", "Bailey",
-           role="aut"),
-    person("Gabriel J.", "Odom",
-           role="aut"),
-    person("Thilde", "Terkelsen",
+  c(person("Astrid", "Saksager",
            role="aut"),
     person("Mona", "Nourbakhsh",
            role="aut"),
-    person("Astrid", "Saksager",
-           role="aut"),
     person("Nikola", "Tom",
-	   role="aut"), 	
-    person("Tiago C.", "Silva",
-           role="aut"),
-    person("André V.", "Olsen",
-           role="aut"),
-    person("Laura", "Cantini",
-           role="aut"),
-    person("Andrei", "Zinovyev",
-           role="aut"),
-    person("Emmanuel", "Barillot",
-           role="aut"),
-    person("Houtan", "Noushmehr",
-           role="aut"),
-    person("Gloria", "Bertoli",
-           role="aut"),
-    person("Isabella", "Castiglioni",
-           role="aut"),
-    person("Claudia","Cava",
-           role="aut"),
-    person("Gianluca", "Bontempi",
            role="aut"),
     person("Xi Steven", "Chen",
            role="aut"),
-    person("Elena", "Papaleo",
+    person("Antonio", "Colaprico",
+           role="aut"),
+    person("Catharina", "Olsen",
            role="aut"),
     person("Matteo", "Tiberti",
            role=c("cre", "aut"),
-           email="tiberti@cancer.dk",
-           ))
-Depends: R (>= 3.5), doParallel, foreach
+	   email="tiberti@cancer.dk"),
+    person("Elena", "Papaleo",
+	   role="aut",
+    	   ))
+Depends: R (>= 4.2), doParallel, foreach
 Imports: parmigene, randomForest, SummarizedExperiment, gplots,
         circlize, RColorBrewer, HiveR, clusterProfiler, DOSE, Biobase,
-        limma, grDevices, graphics, TCGAbiolinks, GEOquery, stats,
+        limma, grDevices, graphics, TCGAbiolinks (>= 2.25.3), GEOquery, stats,
         RISmed, grid, utils, ComplexHeatmap, GenomicRanges, dplyr, fuzzyjoin,
         rtracklayer, magrittr, qpdf, readr, seqminer, stringr,
         tibble, tidyHeatmap, tidyr
 Description: Motivation: The understanding of cancer mechanism requires
```

Comment:
here I would remove the split between Motivation and Results. This should be the description **of the R package** and not much a paper abstract